### PR TITLE
removed designer note

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

**Review by mikehoffms — APPROVED** [review]

**Comment by vchapel** [comment]
Line 18 "Ensure you install the following list of pre-requisites before proceeding." would read better as "Install the following list of pre-requisites before proceeding."

**Comment by vchapel** [comment]
I found a couple more super minor things ... 13 x "ensure" many would read better as "make sure" and on line 161/158 a "now, " that should just be "now"

**Comment by vchapel** [comment]
Next steps line 331/328 is wordy and redundant when you read the intro sentence in combination with any of the bullet points,

**Comment by mikehoffms** [comment]
Re: Victor's comments about lines that aren't changed by this PR:
A recent policy for the Docs pipeline has been defined: 
- If an incoming PR from a SME is a spot-fix, do not edit the entire .md file. Only do a spot-edit on the changed lines.
Therefore this file needs an edit pass in a different PR, such as a global Edit pass of .md files.

**Comment by jasonstephen15** [comment]
@mikehoffms good to merge

File: microsoft-edge/webview2/get-started/winforms.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ In this article, get started creating your first WebView2 app and learn about th
 
 ## Prerequisites  
 
-Ensure you install the following list of pre-requisites before proceeding.  
+Install the following list of pre-requisites before proceeding.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/get-started/winforms.md:18]:
```suggestion
Install the following list of prerequisites before proceeding.
```

**Pull request merged**